### PR TITLE
Fixed hard code for support mail 

### PR DESCRIPTION
**Pull request by jchen293**
Added a variable for support mail and replaced it with hard-coded emails with the constant inline with the strings using it.
Removed print statement.

### CODE REVIEW

File: src/test/java/com/easypost/EasyPostTest.java [hunk 1]
```diff
@@ -851,15 +851,6 @@ public void testCustomsInfo() throws EasyPostException {
...
     shipment = shipment.buy(shipment.lowestRate(buyCarriers));
-
-    ///System.out.println("Shipment.getForms(): >>"+shipment.getForms()+"<<");
-    /// java.lang.IndexOutOfBoundsException: Index: 0, Size: 0
-    /// Form does not exists. Invalid test
-    /*
-    assertTrue(
-        "Customs Form was not submitted electronically",
-        shipment.getForms().get(0).getSubmittedElectronically());
-        */
```
Comment (Justintime50) [inline src/test/java/com/easypost/EasyPostTest.java:862]: Let's leave code alone that isn't part of the ask for the PR.

File: src/main/java/com/easypost/net/EasyPostResponse.java [hunk 2]
```diff
@@ -8,8 +8,6 @@
 	public EasyPostResponse(int responseCode, String responseBody) {
 		this.responseCode = responseCode;
 		this.responseBody = responseBody;
-
-    // System.out.println(this.responseBody);
```
Comment (Justintime50) [inline src/main/java/com/easypost/net/EasyPostResponse.java:12]: This is already being removed from the JSON PR @nwithan8 is doing, you can leave it alone and it'll be take care of separately.

File: src/main/java/com/easypost/net/EasyPostResource.java [hunk 3]
```diff
@@ -583,17 +584,12 @@ public String getId() {
 	public String getMode() {
 		return "";
 	}
-	// Batch
 	public List<Shipment> getShipments() {
 		return new ArrayList<Shipment>();
 	}
-	// public BatchStatus getStatus() {
-	// 	return new BatchStatus();
-	// }
```
Comment (Justintime50) [inline src/main/java/com/easypost/net/EasyPostResource.java:592]: Let's leave code alone that isn't part of the ask for the PR.

**Review by Justintime50 — CHANGES_REQUESTED** [review]

**Review by nwithan8 — APPROVED** [review]
LGTM

**Review by Justintime50 — COMMENTED** [review]
Ah yes... string interpolation - one of many things Java doesn't have. Would we want to consider using `String.format()` instead to clean it up so we don't have to break up the strings?

**Comment by jchen293** [comment]
Used String.format() to clean up the print statement.

File: src/main/java/com/easypost/model/EventDeserializer.java [hunk 4]
```diff
@@ -71,7 +71,7 @@ private Object deserializeJsonElement(JsonElement element) {
 			return deserializeJsonArray(element.getAsJsonArray());
 		} else {
 			System.err.println("Unknown JSON element type for element " + element + ". " +
-					"Please email us at support@easypost.com.");
+					"Please email us at " + EasyPostResource.EASYPOST_SUPPORT_EMAIL + ".");
```
Comment (Justintime50) [inline src/main/java/com/easypost/model/EventDeserializer.java:74]: Looks like there was one missed to switch to `String.format()`

**Review by Justintime50 — CHANGES_REQUESTED** [review]
The rest looks good, one was missed.

**Review by Justintime50 — APPROVED** [review]

**Review by nwithan8 — APPROVED** [review]

**Pull request merged**